src/Parser/Core/Modules/Features/SpellInfo.js

```diff
@@ -0,0 +1,38 @@
+import SPELLS from 'common/SPELLS';
+
+/*
+ * Fields:
+ * int: spell scales with Intellect
+ * crit: spell scales with (is able to or procced from) Critical Strike
+ * hasteHpm: spell does more healing due to Haste, e.g. HoTs that gain more ticks
+ * hasteHpct: spell can be cast more frequently due to Haste, basically any spell except for non haste scaling CDs
+ * mastery: spell is boosted by Mastery
+ * masteryStack: spell's HoT counts as a Mastery Stack
+ * vers: spell scales with Versatility
+ * multiplier: spell scales with whatever procs it, should be ignored for purpose of weights and for 'total healing' number
+ * ignored: spell should be ignored for purpose of stat weights
+ */
+
+// This only works with actual healing events; casts are not recognized.
+export default {
+  [SPELLS.LIGHTS_EMBRACE_HEALING.id]: { // Sea Star of the Depthmother
+    int: false,
+    crit: true,
+    hasteHpct: true, // until LoD's CD is below 8 sec, this speeds up the deck cycle time
```

Comment:
What's LoD?

Reply:
Light of Dawn. I don't think we can currently stop it scaling with haste when it passes x%

Reply:
What does Light of Dawn have to do with Sea Star though? Is this like a generalized case that also applies to other spells?

Reply:
it procs sea star

Reply:
And sea star has an icd